TicTacToe/Board.cs

```diff
@@ -0,0 +1,39 @@
+namespace TicTacToe
+{
+    public class Board
+    {
+        private int[,] _boardArray;
+        
+        public Board()
+        {
+            _boardArray = new[,]
+            {
+                {0, 0, 0},
+                {0, 0, 0},
+                {0, 0, 0}
+            };
+        }
+
+        public int[,] GetBoard()
+        {
+            return _boardArray;
+        }
+
+        public void PlaceMarker(Move move, int player)
+        {
+            var (coordX, coordY) = move.GetCoord();
```

Comment:
I think having getters on your _coordX and _coordY would be better, you could then pass that straight to your board.